src/Microsoft.Sbom.Adapters/Adapters/ComponentDetection/RubyGemsComponentExtensions.cs

```diff
@@ -24,9 +24,11 @@ public static SbomPackage ToSbomPackage(this RubyGemsComponent rubyGemsComponent
         PackageName = rubyGemsComponent.Name,
         PackageVersion = rubyGemsComponent.Version,
         PackageSource = rubyGemsComponent.Source,
-        LicenseInfo = string.IsNullOrWhiteSpace(component.LicenseConcluded) ? null : new LicenseInfo
+        Supplier = string.IsNullOrEmpty(component.Supplier) ? null : $"Person: {component.Supplier}",
```

Comment:
Is there a standard we're trying to follow here? `Person` implies to me that there's an individual human on the right side of the `:`, when it's more likely that it's a corporation or other entity.